src/software/jetson_nano/services/network.h

```diff
@@ -53,4 +53,10 @@ class NetworkService
     // Functions to callback primitiveSet and world and stores them in a variable
     void primitiveSetCallback(TbotsProto::PrimitiveSet input);
     void worldCallback(TbotsProto::World input);
+
+    double last_primitive_time = 0.0;
+    double last_world_time     = 0.0;
```

Comment:
Neither of these two values are really being used. We probably should remove them and implement something like what we have for lost primitive sets. Could be a good level 3 task!